lib/restforce_mock/sandbox.rb

```diff
@@ -12,8 +12,26 @@ def add_object(name, id, values)
       RestforceMock::Sandbox.add_object(name, id, values)
     end
 
-    def self.get_object(name, id)
-      storage[name][id]
+    def get_object(name, id)
```

Comment:
this is confusing, so `id` here can either be an actual `id` or a `query`? does it have to be one method that has to handle both cases? why not separate methods?